interface/therapy_groups/therapy_groups_views/listGroups.php

```diff
@@ -264,6 +265,19 @@
         });
     });
 
+    function oeFormatShortDate(value){
```

Comment:
btw, if you think it would be useful to have this function in other places in the future, I'd be happy to turn this function into a separate script (oeformatshortdate_js.js.php).